Doc/whatsnew/3.12.rst

```diff
@@ -475,7 +475,7 @@ Porting to Python 3.12
   copied as-is to the result string, and any extra arguments discarded.
   (Contributed by Serhiy Storchaka in :gh:`95781`.)
 
-* Fixed wrong sign placement in :c:func:`PyUnicode_FromFormat` and 
```

Comment:
Sorry!

Reply:
Oh, no problem!  I'm just not a fan of the nitpycky tests :)